modules/op-tkn-pipeline-resource-management.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// *  cli_reference/tkn_cli/op-tkn-reference.adoc
+
+[id="op-tkn-pipeline-resource-management_{context}"]
+= Pipeline resource management commands
+
+== tkn resource
+Manage pipeline resources.
+
+.Example: Display help
+----
+$ tkn resource -h
+----
+
+== tkn resource create
+Create a pipeline resource.
+
+.Example: Create pipeline resource defined by the `myresource.yaml` file in a namespace
+----
+$ tkn resource create -f myresource.yaml -n myspace
+----
+
+== tkn resource delete
+Delete a pipeline resource.
+
+.Example: Delete the `myresource` pipeline resource from a namespace
+----
+$ tkn resource delete myresource -n myspace
+----
+
+== tkn resource describe
+Describes a pipeline resource.
```

Comment:
don't use the verb in third-person

Reply:
@ppitonak we have tried to keep the style consistent with the OCP docs: https://docs.openshift.com/container-platform/4.3/cli_reference/openshift_cli/developer-cli-commands.html. Also we have to use active voice. In this case it should be Describe instead of Describes.